prophecies/apps/frontend/src/views/UserRetrieveBookmarks.vue

```diff
@@ -57,7 +57,7 @@ export default {
         const title = this.$t(
           'userRetrieveBookmarks.unableToRetrieveTheBookmarks'
         )
-        this.$router.replace({ name: 'error', params: { title, error } })
+        // this.$router.replace({ name: 'error', params: { title, error } })
```

Comment:
Should this line disappear ?
```suggestion
```

Reply:
Hmmm, no it shouldn't be commented. Thanks for spotting it!